rllib/policy/eager_tf_policy.py

```diff
@@ -179,27 +180,30 @@ def apply_gradients(self, grads):
     return TracedEagerPolicy
 
 
-def build_eager_tf_policy(name,
-                          loss_fn,
-                          get_default_config=None,
-                          postprocess_fn=None,
-                          stats_fn=None,
-                          optimizer_fn=None,
-                          gradients_fn=None,
-                          apply_gradients_fn=None,
-                          grad_stats_fn=None,
-                          extra_learn_fetches_fn=None,
-                          extra_action_fetches_fn=None,
-                          validate_spaces=None,
-                          before_init=None,
-                          before_loss_init=None,
-                          after_init=None,
-                          make_model=None,
-                          action_sampler_fn=None,
-                          action_distribution_fn=None,
-                          mixins=None,
-                          obs_include_prev_action_reward=True,
-                          get_batch_divisibility_req=None):
+def build_eager_tf_policy(
+        name,
+        loss_fn,
+        get_default_config=None,
+        postprocess_fn=None,
+        stats_fn=None,
+        optimizer_fn=None,
+        gradients_fn=None,
```

Comment:
Changes to this also need to be reflected in documentation right?

Reply:
Changed it everywhere in the docs and remaining RLlib code.
The soft deprecation (warning) should explain to users what to do.
Reason I changed this is to make torch and tf policy building even more similar.